### PR TITLE
chore(showcase): Add peluqueriavenezolana.cl

### PR DESCRIPTION
**Pull request by enBonnet**
## Description

add peluqueriavenezolana.cl to showcase

### CODE REVIEW

**Comment by enBonnet** [comment]
Thanks @lannonbr

**Comment by enBonnet** [comment]
Hey @LekoArts could you help me with this? 😄

**Comment by enBonnet** [comment]
Hello @pieh could you help me here?

File: docs/sites.yml [hunk 1]
```diff
@@ -8334,3 +8334,14 @@
   built_by: Peter Brady
   built_by_url: https://www.peterbrady.co.uk
   featured: false
+- title: Salexa - Estetica Venezolana
+  url: https://peluqueriavenezolana.cl/
+  main_url: https://peluqueriavenezolana.cl/
+  source_url: https://github.com/enbonnet/salexa-front
+  description: >
+    Sitio de una estetica o peluqueria de venezolanos en Chile
```
Comment (pieh) [inline docs/sites.yml]: Please use english description for our showcase

Comment (enBonnet) [reply docs/sites.yml]: Already update this, thanks for the correction

File: docs/sites.yml [hunk 2]
```diff
@@ -8334,3 +8334,14 @@
   built_by: Peter Brady
   built_by_url: https://www.peterbrady.co.uk
   featured: false
+- title: Salexa - Estetica Venezolana
+  url: https://peluqueriavenezolana.cl/
+  main_url: https://peluqueriavenezolana.cl/
+  source_url: https://github.com/enbonnet/salexa-front
+  description: >
+    Venezuelan beauty or hairdressing site in Chile
```
Comment (pieh) [inline docs/sites.yml]: Should it be
```suggestion
    Venezuelan beauty and hairdressing salon in Chile
```
?

Comment (enBonnet) [reply docs/sites.yml]: Yes, the suggestion is good, my confusion was to talk about the place and not the website, now it's better

**Pull request merged**